dcli/language/kotlin.py

```diff
@@ -0,0 +1,32 @@
+from .language import Language
+
+TEMPLATE_PATH = '/src/main/kotlin/template'
+SUCCESS_PATH = '/src/main/kotlin/success'
+APP_PATH = '/src/main/kotlin/app'
+
+SOLVE_PATH = '/src/main/kotlin/app/Solve.kt'
+RUN_PATH = '/src/main/kotlin/app/Run.kt'
+
+ASSET_PATHS = ['/src/main/kotlin/app/Logger.kt']
+
+TARGET_FILE = 'Target'
+
+class Kotlin(Language):
+    def __init__(self):
+        Language.__init__(self,
+            'kotlin',
+            TEMPLATE_PATH,
+            SUCCESS_PATH,
+            APP_PATH,
+            SOLVE_PATH,
+            RUN_PATH,
+            TARGET_FILE,
+            'kotlin',
```

Comment:
This argument is the extension, means you have to set `.kt`, see example in `python.py`

Reply:
\